prelude-nri/elm/isolated_compile.py

```diff
@@ -22,7 +22,7 @@ def symlink_if_necessary(source, target):
      - If the target already exists and points somewhere else, remove it
        and relink.
     """
-    if os.path.exists(target):
+    if os.path.lexists(target):
```

Comment:
Didn't know the different between lexists vs exists, but makes sense for this case, nice catch!

Reply:
I didn’t either, until it failed for a broken symlink locally. 😆